ocpmodels/trainers/forces_trainer.py

```diff
@@ -76,42 +85,80 @@ def __init__(
 
     def load_task(self):
         print("### Loading dataset: {}".format(self.config["task"]["dataset"]))
-        self.dataset = registry.get_dataset_class(
-            self.config["task"]["dataset"]
-        )(self.config["dataset"])
-        num_targets = 1
-
-        self.num_targets = num_targets
-        (
-            self.train_loader,
-            self.val_loader,
-            self.test_loader,
-        ) = self.dataset.get_dataloaders(
-            batch_size=self.config["optim"]["batch_size"]
-        )
+
+        if "val_dataset" in self.config:
+            self.train_dataset = registry.get_dataset_class(
+                self.config["task"]["dataset"]
+            )(self.config["dataset"])
+            self.val_dataset = registry.get_dataset_class(
+                self.config["task"]["dataset"]
+            )(self.config["val_dataset"])
+
+            self.train_loader = DataLoader(
+                self.train_dataset,
+                batch_size=self.config["optim"]["batch_size"],
+                shuffle=True,
+                collate_fn=data_list_collater,
+                num_workers=self.config["optim"]["num_workers"],
+            )
+            self.val_loader = DataLoader(
+                self.val_dataset,
+                batch_size=1,
+                shuffle=False,
+                collate_fn=data_list_collater,
+                num_workers=self.config["optim"]["num_workers"],
+            )
+            self.test_loader = None
+        else:
+            self.dataset = registry.get_dataset_class(
+                self.config["task"]["dataset"]
+            )(self.config["dataset"])
+            (
+                self.train_loader,
+                self.val_loader,
+                self.test_loader,
+            ) = self.dataset.get_dataloaders(
+                batch_size=self.config["optim"]["batch_size"]
+            )
+
+        self.num_targets = 1
 
         # Normalizer for the dataset.
         # Compute mean, std of training set labels.
         self.normalizers = {}
         if self.config["dataset"].get("normalize_labels", True):
```

Comment:
We should make this default False since normalization parameters are now required to be specified, or at least throw an error that those aren't provided.